src/rules/completedDocsRule.ts

```diff
@@ -499,6 +539,14 @@ function walk(context: Lint.WalkContext<ExclusionsMap>) {
             );
         }
 
+        if (tsutils.isConstructorDeclaration(node) && tsutils.isClassDeclaration(node.parent)) {
```

Comment:
```suggestion
        if (tsutils.isConstructorDeclaration(node)) {
```

There's no case I can think of where we'd care what the parent of a constructor node is. Class declarations and class expressions are both valid class types.

Reply:
Done